parkcore_app/pubspec.yaml

```diff
@@ -44,6 +44,8 @@ dependencies:
   uuid: 2.0.4
 
 dev_dependencies:
+  pedantic: ^1.4.0
```

Comment:
should this be in dev_dependencies? or dependencies?

Reply:
I took this off the instructions of putting google maps on flutter